globus_sdk/nexus.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import print_function
+
+from globus_sdk.base import BaseClient
+
+
+class NexusClient(BaseClient):
+    def __init__(self, environment="default"):
+        BaseClient.__init__(self, "nexus", environment)
+
+    def get_goauth_token(self, username, password):
```

Comment:
Agreed but we should wait until 2-legged or possibly API keys/app passwords are available as a replacement. Otherwise we'll get a lot of questions about it, and have not alternative to provide them.